### DISCUSSION TITLE
Minifig can build code

### DISCUSSION THREAD

**Discussion by GreatBrickLab** [rgriebl/brickstore#906] [Ideas]
I have the code that you could use to allow anyone to evaluate their store to see that minifigs they can build based on the current inventory.    I feel like it could be run right from the application with some variables it would not take long for most stores,  

UPDATE:  since posting this I do see that the “can build” is already available for sets and for figs.   

With a large inventory like ours it would help to find a fig or set in the can build list and then be able to part out the fig with the remarks of where the parts are located.  And the correct qty.   

If you do start working on this feature the can build would be more useful with a % slider.  For what % to complete a set could be built to.

**Comment by StoneyBricks** [comment]
That would be a great addition.
